modules/distribution/src/repository/resources/dbscripts/apimgt/mssql.sql

```diff
@@ -272,7 +274,7 @@ IF NOT  EXISTS (SELECT * FROM SYS.OBJECTS WHERE OBJECT_ID = OBJECT_ID(N'[DBO].[S
 CREATE TABLE SP_AUTH_STEP (
             ID INTEGER NOT NULL IDENTITY,
             TENANT_ID INTEGER NOT NULL,
-            STEP_ORDER INTEGER DEFAULT 1,
+       	    STEP_ORDER INTEGER DEFAULT 1,
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04